app/controllers/api/v1/stop_stations_controller.rb

```diff
@@ -0,0 +1,33 @@
+# == Schema Information
+#
+# Table name: current_stops
+#
+#  id                                 :integer          not null, primary key
+#  onestop_id                         :string
+#  geometry                           :geography({:srid geometry, 4326
+#  tags                               :hstore
+#  created_at                         :datetime
+#  updated_at                         :datetime
+#  name                               :string
+#  created_or_updated_in_changeset_id :integer
+#  version                            :integer
+#  identifiers                        :string           default([]), is an Array
+#  timezone                           :string
+#  last_conflated_at                  :datetime
+#  type                               :string
+#  parent_stop_id                     :integer
+#
+# Indexes
+#
+#  #c_stops_cu_in_changeset_id_index      (created_or_updated_in_changeset_id)
+#  index_current_stops_on_geometry        (geometry)
+#  index_current_stops_on_identifiers     (identifiers)
+#  index_current_stops_on_onestop_id      (onestop_id)
+#  index_current_stops_on_parent_stop_id  (parent_stop_id)
+#  index_current_stops_on_tags            (tags)
+#  index_current_stops_on_updated_at      (updated_at)
+#
+
+class Api::V1::StopStationsController < Api::V1::StopsController
```

Comment:
I imagine the stop_stations endpoint will need different/more query params than the stops endpoint. Say, querying for the stop_stations that include a certain platform.

Not necessary to squeeze into this initial PR, but let's talk more in our call tomorrow with @gknisely and other about the range of possible query parameters for filtering and sorting.